src/System.IO.FileSystem/src/System/IO/Win32FileSystemObject.cs

```diff
@@ -64,7 +64,7 @@ public DateTimeOffset CreationTime
                 {
                     EnsureDataInitialized();
                     long dt = ((long)(_data.ftCreationTimeHigh) << 32) | ((long)_data.ftCreationTimeLow);
-                    return DateTimeOffset.FromFileTime(dt);
+                    return DateTimeOffset.FromFileTime(dt).ToLocalTime();
```

Comment:
Why is ToLocalTime necessary?  Doesn't FromFileTime already convert to local time?